app/src/main/java/org/sopt/dosopttemplate/ui/doandroid/DoAndroidFragment.kt

```diff
@@ -0,0 +1,32 @@
+package org.sopt.dosopttemplate.ui.doandroid
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.sopt.dosopttemplate.databinding.FragmentDoAndroidBinding
+
+class DoAndroidFragment : Fragment() {
+    private var _binding: FragmentDoAndroidBinding? = null
+    private val binding: FragmentDoAndroidBinding
+        get() = requireNotNull(_binding)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View? {
```

Comment:
위에서 null check를 해줬으니
```suggestion
    ): View {
```
도 좋아보입니다.